pom.xml

```diff
@@ -133,19 +132,20 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <source>1.7</source>
-                    <doclet>org.asciidoctor.Asciidoclet</doclet>
+                    <source>11</source>
+                    <additionalJOption>-J--add-exports=jdk.javadoc/jdk.javadoc.internal.tool=ALL-UNNAMED</additionalJOption>
```

Comment:
Since this is potentially being used in the README, it should probably get a call out to explain why this is needed (if it is for consumers)